src/Modules/OrchardCore.Commerce.ContentFields/Settings/AddressFieldSettingsDriver.cs

```diff
@@ -9,17 +10,14 @@ namespace OrchardCore.Commerce.Settings;
 
 public class AddressFieldSettingsDriver : ContentPartFieldDefinitionDisplayDriver<AddressField>
 {
-    public override IDisplayResult Edit(ContentPartFieldDefinition model) =>
-        Initialize<AddressPartFieldSettings>("AddressFieldSettings_Edit", viewModel =>
-            {
-                var settings = model.Settings.ToObject<AddressPartFieldSettings>();
-                viewModel.Hint = settings.Hint;
-            })
-            .PlaceInContent();
+    public override IDisplayResult Edit(ContentPartFieldDefinition model, BuildEditorContext context)
+        => Initialize<AddressPartFieldSettings>("AddressFieldSettings_Edit", viewModel =>
```

Comment:
One arrow is after the line break, the other is before. This feels very random. Why?
Please move all `=>` at the end of the line instead of the beginning.
```suggestion
    public override IDisplayResult Edit(ContentPartFieldDefinition model, BuildEditorContext context) =>
        Initialize<AddressPartFieldSettings>("AddressFieldSettings_Edit", viewModel =>
```